### DISCUSSION TITLE
有考虑支持多项目编译和管理的功能支持

### DISCUSSION THREAD

**Discussion by zjy2931** [NervJS/taro#15232] [Q & A]
### 这个特性解决了什么问题？
不知道有多少人 遇到同一个业务线承担多个小程序项目 每次修改都是copy  而且小程序的ui风格和用户体系都是一套 维护起来真的费劲

### 这个 API 长什么样？
如果能抽取公共逻辑  多个小程序打包分别输出就好了

**Comment by TomVista** [comment]
https://taro-docs.jd.com/taro/docs/template

**Comment by yymzy** [comment]
https://www.npmjs.com/package/taro-ci  支持同一套代码打包不同的微信或支付包小程序，并支持ci上传
https://www.npmjs.com/package/taro-plugin-build 配合此插件使用，可以实现各小程序之间不同模块的维护
